build.gradle

```
@@ -1,12 +1,12 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.5.0"
+    ext.kotlin_version = '1.5.21'
```

Let's update this to the latest version

```suggestion
    ext.kotlin_version = '1.7.20'
```